library.properties

```diff
@@ -0,0 +1,10 @@
+name=ESPInfluxdb
+version=1.0.0
+author=hwwong
+maintainer=hwwong
+sentence=ESP32 / ESP8266 influxdb client.
+paragraph=ESP32 / ESP8266 influxdb client, posts and queries data
+category=Data Storage
+url=https://github.com/hwwong/ESP_influxdb
+architectures=ESP8266,ESP32
```

Comment:
Architectures value is case sensitive and the correct case of these architecture values is esp8266 and esp32. The current values causes the example to appear under **File > Examples > INCOMPATIBLE** and a warning to be shown when compiling the library:
```
WARNING: library ESP_influxdb claims to run on (ESP8266, ESP32) architecture(s) and may be incompatible with your current board which runs on (esp8266) architecture(s).
```